src/hooks/useTelegramMock.ts

```diff
@@ -1,18 +1,18 @@
-import { useClientOnce } from '@/hooks/useClientOnce';
 import {
   isTMA,
   type LaunchParams,
   mockTelegramEnv,
   parseInitData,
   retrieveLaunchParams,
-} from '@telegram-apps/sdk-react';
+} from "@telegram-apps/sdk-react";
```

Comment:
Single quotes were changed to double quotes in quite a few code lines. Probably, IDE formatted those lines. Maybe these changes could be reverted, since they don't bring actual changes.
I think adding prettier config to this repository could help with this later.